Incorrect PDB file written for atoms.

* molsystem version: 2021.11.15
* Python version: 3.9
* Operating System: MacOs

### Description

Running PACKMOL step for Ar ... failed reading the PDB input file because of a `connect` record.

### What I Did

The packmol_step flowchart test failed.
```
=========================== short test summary info ============================
FAILED tests/test_flowcharts.py::test_ideal_gas[/home/runner/work/packmol_step/packmol_step/tests/flowcharts/packmol-ar.flow]
========================= 1 failed, 25 passed in 4.47s =========================

```

--- a/molsystem/pdb.py
+++ b/molsystem/pdb.py
@@ -237,7 +237,8 @@ class PDBMixin:
 
         # bonds
         for i, js in enumerate(self.bonded_neighbors(as_indices=True)):
-            lines.append(f"CONECT{i+1:5d}" + "".join([f"{j+1:5d}" for j in js]))
+            if len(js) > 0:
+                lines.append(f"CONECT{i+1:5d}" + "".join([f"{j+1:5d}" for j in js]))
 
         lines.append(
             "MASTER        0    0    0    0    0    0    0    0"

